### PR TITLE
Added a port table on the index page

### PR DESCRIPTION
**Pull request by ohlookadollar**

### CODE REVIEW

**Comment by radaeze** [comment]
@ohlookadollar Thanks!

File: index.html.md.erb [hunk 1]
```diff
@@ -96,6 +96,35 @@ The following table provides version and version-support information about
...
+    <td>28893</td>
+  </tr>
+  <tr>
+    <td>VM with the scanner</td>
+    <td>The port is bound to the loopback interface, the scanning server receiving the request passes the request through to the RPC server via the port doing the actual scan</td>
+    <td>28895</td>
+  </tr>
+</table>
+
+* Consider whitelisting the syslog forwarding port, and the aws/minio/azure bucket ports on the oscap_store if those feature are enabled.
```
Comment (xtreme-nitin-ravindran) [inline index.html.md.erb]: aws/minio/azure bucket port on the oscap_store??

Comment (DennyHoang) [reply index.html.md.erb]: True, that doesnt make sense. It is external connection so the port would be on AWS/Minio/Azure side.

Comment (Pivotal-Christopher-Wong) [reply index.html.md.erb]: So is there going to be an adjustment to this then?

**Pull request merged**